bin/deploy

```diff
@@ -5,82 +5,98 @@ BIN_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
 function usage {
   cat <<EOS
-Usage: ./$SCRIPT_NAME <connection-string> <environment>
- e.g.: ./$SCRIPT_NAME core@api.biketag-staging.jackpine.me staging
+Usage: $SCRIPT_NAME <connection-string> <environment>
+ e.g.: $SCRIPT_NAME core@api.biketag-staging.jackpine.me staging
+
+You can deploy against a local docker-machine, but make sure you have the auth keys
+ e.g.: ssh-add ~/.docker/machine/machines/default/id_rsa
+ e.g.: $SCRIPT_NAME docker@\`docker-machine ip\` development
 EOS
 }
 
 # Must be absolute path
```

Comment:
Remove this comment?  Or do you consider `~/` to be an absolute path?  I am on the fence about this distinction.  As we deploy on more diverse environments (linux, windows, and macos), I have been replacing `~/` with `${HOME}`.